modules/govuk/templates/etc/cron.d/snapshot.erb

```diff
@@ -0,0 +1 @@
+*/10 0-4,6-23 * * * <%= @snapshot_user %> /usr/local/bin/rummager-snapshot
```

Comment:
Sorry, I misread the cron timing.

I found [this](http://crontab.guru/#*/10_0-4,6-23_*_*_*) helpful to interpret it. It runs at every hour except for at 5am.

So if the pruning and snapshotting runs at the same time, Elasticsearch becomes unresponsive? I think a lock would be a good idea as relying on timing is liable to someone changing times elsewhere and not understanding this dependency.